jupyterhub/handlers/base.py

```diff
@@ -388,6 +390,7 @@ def active_server_limit(self):
     @gen.coroutine
     def spawn_single_user(self, user, server_name='', options=None):
         # in case of error, include 'try again from /hub/home' message
+        spawn_starttime = time.perf_counter()
```

Comment:
Let's rename to `spawn_start_time` since it's less error prone than the double `tt`

Reply:
Done!